### PR TITLE
Multicore implementation

### PR DESCRIPTION
**Pull request by Epiczhul**
We now finaly have MADT and it works! It detects all CPU cores.
image

### CODE REVIEW

**Comment by 9xbt** [comment]
What the fuck is this. You removed the shutdown code and most of the code is ChatGPT generated

File: source/Cosmos.Core/ACPI.cs [hunk 1]
```diff
@@ -188,6 +263,44 @@ static bool Check_RSD(uint address)
...
+                            // Here you can add this APIC ID to a list or process it as needed.
+                        }
+                        break;
+
+                    case MADTEntryType.IoApic:
+                        IoApicEntry* ioApic = (IoApicEntry*)entryPtr;
+                        Console.WriteLine($"Found IO APIC with ID: {ioApic->IoApicId} at address: {ioApic->IoApicAddress:X}");
+                        break;
+
+                        // Handle other MADT entries if needed
```
Comment (winksplorer) [inline source/Cosmos.Core/ACPI.cs:297]: You said "I MADE MADT WORK"
But these types of comments are usually from ChatGPT. Not judging, I use ChatGPT code too sometimes, but it's still not your code, stop pretending it is.

Comment (Epiczhul) [reply source/Cosmos.Core/ACPI.cs:297]: No I Added the Comments because Last Time I comitted something they hated me because I removed all comments

Comment (Epiczhul) [reply source/Cosmos.Core/ACPI.cs:297]: Only the Comments are Copilot Not the Code

Comment (winksplorer) [reply source/Cosmos.Core/ACPI.cs:297]: you didn't ask copilot to add comments, you asked it something like "add madt" and gave it the ACPI.cs. "// Handle other MADT entries if needed" does not come from an AI adding comments separately.

**Pull request closed without merging**